src/odemis/gui/comp/viewport.py

```diff
@@ -1103,7 +1103,7 @@ def _on_new_data(self, data):
         Called when a new data is available (in a live stream)
         data (1D DataArray)
         """
-        if data.size:
+        if data is not None:
```

Comment:
I've also fixed that in my PR about the viewport. I'd suggest you drop it here.

Reply:
Fixed what exactly?